documentation/docs/concepts/decentralized_identifiers/resolve.mdx

```diff
@@ -6,187 +6,148 @@ image: /img/Identity_icon.png
+
+// Configure a client for the Shimmer testnet "rms".
+const nodeUrl = "https://api.testnet.shimmer.network/";
+const client = new Client({
+    primaryNode: nodeUrl,
+    localPow: true,
+});
+const didClient = new StardustIdentityClient(client);
+
+// Parse the DID and resolve its DID Document.
+const did = StardustDID.parse("did:iota:rms:0x7b48b06232b8a1e7a31c314cab1ceedb84e2e9dd2b1fae79b67eaa4595f15e47");
+const document = await didClient.resolveDid(did);
 ```
+
 </TabItem>
 </Tabs>
 
+## Resolution for Verifiable Presentations
 
-## Complete examples
-This section shows complete examples from the Iota Identity Framework code base. The first example creates a DID Document, publishes it to the Tangle and then resolves it.
-
-<CodeSnippet
-    nodeReplitLink="https://repl.it/@IOTAFoundation/resolve-did?lite=true"
-    rustContent={resolve_did_rs}
-    nodeGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/bindings/wasm/examples/src/resolve_did.js"
-    rustGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/examples_legacy/low-level-api/resolve_did.rs"
-/>
-
-
-This second example demonstrates creating, publishing changes and then resolving the history of a DID Document.
-<Tabs
-  groupId="programming-languages"
-  defaultValue="nodejs"
-  values={[
-    { label: "Node.js", value: "nodejs" },
-    { label: "Rust", value: "rust" }
-  ]}
->
-  <TabItem value="nodejs">
-    <CodeBlock className="language-javascript">{resolve_history_js}</CodeBlock>
-  </TabItem>
-  <TabItem value="rust">
-    <CodeBlock className="language-rust">{resolve_history_rs}</CodeBlock>
-  </TabItem>
-</Tabs>
+When validating [verifiable presentations](../verifiable_credentials/verifiable_presentations.mdx), it is necessary to resolve the DID Documents of the [verifiable credential](../verifiable_credentials/overview.md) issuers and presentation holder to verify their signatures. Resolving the necessary DID Documents is performed automatically when verifying presentations via the `Resolver` (see this [example](../verifiable_credentials/verifiable_presentations.mdx#example)).
 
-Note that this example used the `Client` to resolve the history of the DID Document, but one could also use the `Resolver` for this task.
+When direct access to these DID Documents is desired, the `Resolver` also provides standalone methods to:
+- Resolve a presentation holder's DID Document.
+- Resolve the DID Documents of the issuers of the credentials in a verifiable presentation.
+- Resolve the issuer's DID Document for a given verifiable credential.
+
+## Example
+The following example creates a new IOTA DID, publishes it and then resolves its DID Document and Alias Output.
+
+<Tabs>
+    <TabItem value="rust" label="Rust">
+        <CodeBlock className="language-rust">
+            {resolve_did_rs}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="node" label="Node.js">
+        <CodeBlock className="language-javascript">
+            {resolve_did_js}
```

Comment:
This example depends on another file `./ex0_create_did`. A link to GitHub for the example location might be helpful.

Reply:
Preferably we would use the updated `CodeSnippet` once #997 is merged.